cpp/farm_ng/core/logging/logger.cpp

```diff
@@ -51,6 +54,8 @@ void StreamLogger::writeHeader(
           fmt::arg("text", header_text),
           fmt::arg("file", file),
           fmt::arg("line", line),
+          fmt::arg("fileparent", file.substr(parent_idx + 1)),
```

Comment:
Huh, I never used it, but can't you just use: https://en.cppreference.com/w/cpp/filesystem/path/parent_path

Reply:
That provides the entire parent path, not just the name of the parent directory.

Reply:
I agree there's a cleaner way to do this with `std::filesystem` APIs than with `.substr`